float32 teplota
int16 svetlo
int16 zvuk
int16 vzdialenost
